news/pygments-modifications.vendor.rst

```diff
@@ -0,0 +1 @@
+Tree-trim unused portions of vendored pygments, to reduce the distribution size.
```

Comment:
I hope this file would show up after `pygments.vendor.rst`, or the bullets would not read very well in the changelog.

Reply:
https://pip--10462.org.readthedocs.build/en/10462/news/#vendored-libraries

Yup!